### PR TITLE
adds open all button

### PR DESCRIPTION
**Pull request by ablwr**
Long-standing quick hack for the searchability feature request from @bturkus. This button allows for user to open all recipes, so the page can then be searched using browser-based ctrl-f.

### CODE REVIEW

**Comment by retokromer** [comment]
Somewhere we should also give explicitly the CTRL-F hint to make https://github.com/amiaopensource/ffmprovisr/issues/211 possible, I guess.

**Comment by ablwr** [comment]
Oh yes -- seeking feedback on placement of this button and if it should be somewhere else, like on the main section after the info?

**Comment by retokromer** [comment]
I don’t have a strong opinion on the placement.

If I click a second time, nothing new happens. Would it be useful to have an open/close button, instead of a open-only button?

**Comment by ablwr** [comment]
Yes! I will make it into a toggle. Thank you! But does this generally sound
like a good idea?

**Comment by retokromer** [comment]
>But does this generally sound like a good idea?

Yes, it does!

**Comment by privatezero** [comment]
Yah, I definitely think this sounds like a good plan!

**Comment by ablwr** [comment]
OK y'all sorry for being so lazy, I did it right this time!

File: index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@ <h1>➺ ffmprovisr ❥</h1>
   <nav class="sidebar well">
     <h2 class="heading">Table of Contents</h2>
     <a href="#about"><div class="contents-list">About this resource</div></a>
+    <div id="open-all" class="contents-list">Open all recipes</div>
```
Comment (retokromer) [inline index.html]: I would say `Open/close all recipes`.

Comment (JonnyTech) [reply index.html]: `show / hide`?

Comment (ablwr) [reply index.html]: updated with show/hide!

**Review by retokromer — APPROVED** [review]

**Comment by bturkus** [comment]
hey I totally spaced on this, but thanks so much! I'm using it already!

**Pull request merged**